DOC-1567: Updated the editor commands documentation (#2312)
tinymce/tinymce-docs: modules/ROOT/pages/editor-command-identifiers.adoc

[[coreeditorcommands]]
=== Core Editor commands

==== Supported browser-native commands

> The commands on the following table are provided by the {productname} editor and provide the same or similar functionality to the https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#commands[native browser commands].

Reviewer: ```suggestion
The commands in the following table are provided by the {productname} editor and provide the same or similar functionality to the https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#commands[native browser commands].
```

I feel like there's a word that should go before "native browser commands" too... "Respective"? I'm assuming we didn't implement all the native commands, so partly trying to imply it's a subset.

|Indent |Indents the current selection.
|Outdent |Outdents the current selection.
|InsertHorizontalRule |Inserts a horizontal rule at the cursor location or inplace of the current selection.
|InsertLineBreak |Adds a line break `+<br>+` at the current cursor or selection.

> |InsertParagraph |Inserts a new line at the current cursor or selection.

Reviewer 2: We should look into this api in future, perhaps InsertNewLine ?

Author: It needs to use the same name as the native command and can't be changed, as that's why this was implemented (since commands no longer fallback to the native variants).

tinymce.activeEditor.execCommand('BackColor', false, '#FF0000');
tinymce.activeEditor.execCommand('FontName', false, 'courier new');
tinymce.activeEditor.execCommand('FontSize', false, '30px');
tinymce.activeEditor.execCommand('FormatBlock', false, 'bold');
tinymce.activeEditor.execCommand('RemoveFormat');
tinymce.activeEditor.execCommand('Indent');
tinymce.activeEditor.execCommand('Outdent');
tinymce.activeEditor.execCommand('CreateLink', false, 'https://www.tiny.cloud');
tinymce.activeEditor.execCommand('Unlink');
tinymce.activeEditor.execCommand('InsertHorizontalRule');
tinymce.activeEditor.execCommand('InsertParagraph');
tinymce.activeEditor.execCommand('InsertText', false, 'My text content');
tinymce.activeEditor.execCommand('InsertHTML', false, 'My HTML content');
tinymce.activeEditor.execCommand('InsertImage', false, 'https://www.example.com/image.png');
tinymce.activeEditor.execCommand('SelectAll');
tinymce.activeEditor.execCommand('Delete');
tinymce.activeEditor.execCommand('ForwardDelete');
tinymce.activeEditor.execCommand('Redo');
tinymce.activeEditor.execCommand('Undo');
----

==== Additional core editor commands

> The commands on the following table are provided by the {productname} editor and provide additional functionality that do not require any plugins to be enabled.

Reviewer: ```suggestion
The commands in the following table are provided by the {productname} editor and provide additional functionality that does not require any plugins to be enabled.
```
